src/ompl/geometric/planners/experience/src/ThunderRetrieveRepair.cpp

```diff
@@ -148,6 +148,19 @@ namespace ompl
             const base::State *startState = pis_.nextStart();
             const base::State *goalState = pis_.nextGoal(ptc);
 
+            if (startState == nullptr) {
+                OMPL_INFORM("RetrieveRepair::solve() start state pointer is null. Returning "
+                            "PlannerStatus::INVALID_START");
+                return base::PlannerStatus::INVALID_START;
+            }
+
+            if (goalState == nullptr)
+            {
```

Comment:
whitespace: this bracket should go on previous line, but not a huge deal.

Reply:
This repo has a `[.clang-format](https://github.com/DexaiRobotics/ompl/blob/main/.clang-format)` file that is to be used to ensure consistent formatting.